asahi-diagnose

```diff
@@ -67,6 +67,63 @@ system_info() {
 EOF
 }
 
+
+
+# Check for the Pro Audio profile (installed before early 2023)
+check_proaudio() {
+    grep "alsa_output.platform-sound.pro-output.." "${HOME}/.local/state/wireplumber/restore-streams" > /dev/null \
```

Comment:
`.local/state/wireplumber/restore-stream` (without trailing 's')? 

I don't think this is a reliable indication of anything. I think this will match for anyone who played with the profiles for the headphone.

checking `${HOME}/.local/state/wireplumber/default-profile` for `alsa_card.platform-sound=HiFi` would be a useful diagnosis.

Reply:
That file doesn't always exist. The only one that reliably exists is `~/.local/wireplumber/restore-stream`

the plural thing was me copying a static version for debug. should be fixed.

Reply:
the other fact of the matter is that we don't support changing the profile regardless of why it was done. the
[troubleshooting page](https://github.com/AsahiLinux/docs/wiki/Undoing-early-speaker-support-hacks) makes mention of the fact that you could hit this particular issue for a variety of reasons